crates/circuit/src/dot_utils.rs

```diff
@@ -19,7 +19,7 @@ use std::collections::BTreeMap;
 use std::io::prelude::*;
 
 use crate::dag_circuit::{DAGCircuit, Wire};
-use pyo3::prelude::*;
+use pyo3::{prelude::*, IntoPyObjectExt};
```

Comment:
I'm surprised clippy doesn't complain about this, but I find mixing `*` in a single line this really hard to read, it's easy enough to just add a separate `use statement` in cases like this.

```suggestion
use pyo3::prelude::*;
use pyo3::IntoPyObjectExt;
```

Reply:
Done in 239370d0b.